app/controllers/users_controller.rb

```diff
@@ -50,6 +51,18 @@ def destroy
     redirect_to users_url, status: :see_other
   end
 
+  def following
+    @title = t ".following"
+    @pagy, @users = pagy @user.following, items: Settings.page_items
+    render "show_follow", status: :unprocessable_entity
```

Comment:
```suggestion
    render :show_follow, status: :unprocessable_entity
```
Thử như này xem có chạy ko e nhé